src/node2json.js

```diff
@@ -28,8 +28,9 @@ const convertToJson = function(node, options) {
     var tagname = keys[index];
     if (node.child[tagname] && node.child[tagname].length > 1) {
       jObj[tagname] = [];
-      for (var tag in node.child[tagname]) {
-        jObj[tagname].push(convertToJson(node.child[tagname][tag], options));
+      const keys = Object.keys(node.child[tagname]);
```

Comment:
Since conditional above this is checking `node.child[tagname].length > 1` which I assume is a shorthand for checking that `node.child[tagname]` is an array, would it be better to use `for(let tag of node.child[tagname])` instead? This would apply to to `src/node2json_str.js` as well.

Reply:
Yes, I feel `node.child[tagname]` can be used. For `for(let tag of node.child[tagname])`, I would prefer to go with classic for loop style everywhere.